src/HealthChecks.DocumentDb/DocumentDbHealthCheck.cs

```diff
@@ -30,7 +32,15 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
                     documentDbClient = _connections[_documentDbOptions.UriEndpoint];
                 }
             }
-            await documentDbClient.OpenAsync(cancellationToken).ConfigureAwait(false);
+
+            if (CanCheckCollectionInDatabase())

```

Comment:
```suggestion
            if (!string.IsNullOrWhiteSpace(_documentDbOptions.DatabaseName) && !string.IsNullOrWhiteSpace(_documentDbOptions.CollectionName))
```

Reply:
I'm wondering if having all in-line is better, than having a helper method?

Could you share more thoughts, why in-line is better?

For me having a human-readable method name is easier to read, but let's discuss 😄

Reply:
Because logic is obvious and for me it's more easy to read such code without jumping into helper methods.